addons/info/src/components/types/OneOf.js

```diff
@@ -1,7 +1,13 @@
 import React from 'react';
 import { TypeInfo } from './proptypes';
 
-const OneOf = ({ propType }) => <span>{propType.value.map(({ value }) => value).join(' | ')}</span>;
+const isObject = propType => typeof propType.value === 'object';
```

Comment:
Should it rather check that `propType.value` is an array, as we're using an array method?

Reply:
I'm not sure why the `typeof propType.value` returns `object` when array. Better to change to `Array.isArray()`

Reply:
> why the typeof propType.value returns object when array

Because array is an object =)